### ISSUE TITLE
Runs fine for the first time

### ISSUE THREAD

**Issue by schmitzoide** [johnno1962/InjectionApp#2]
The first time I run this (InjectedCount = 1), all runs well, all other consequent attempts fail with the sign code issue. 

```
Error loading /Users/marcelschmitz/Library/Developer/Xcode/DerivedData/winespots-agamduxrctmuerhhpopnmsqgxyta/Logs/iOSInjectionProject/build/Debug-iphonesimulator/InjectionBundle7.bundle/InjectionBundle:  dlopen(/Users/marcelschmitz/Library/Developer/Xcode/DerivedData/winespots-agamduxrctmuerhhpopnmsqgxyta/Logs/iOSInjectionProject/build/Debug-iphonesimulator/InjectionBundle7.bundle/InjectionBundle, 265): no suitable image found.  Did find:
	/Users/marcelschmitz/Library/Developer/Xcode/DerivedData/winespots-agamduxrctmuerhhpopnmsqgxyta/Logs/iOSInjectionProject/build/Debug-iphonesimulator/InjectionBundle7.bundle/InjectionBundle: required code signature missing for '/Users/marcelschmitz/Library/Developer/Xcode/DerivedData/winespots-agamduxrctmuerhhpopnmsqgxyta/Logs/iOSInjectionProject/build/Debug-iphonesimulator/InjectionBundle7.bundle/InjectionBundle'
```

I've code sign the bundle project, hence working for the first time. Don't know how to sign the consequent bundles. 

Any thoughts?

**Comment by johnno1962** [comment]
Stange problem. Can you delete the injection project and try again without changing the codesigning. It should be fine ad-hoc. I can TeamView if you want me to take a look.

**Comment by schmitzoide** [comment]
Let me try that and come back to you. Really, teamviewer? You're awesome!

**Comment by schmitzoide** [comment]
Guess that didn't work :(

**Comment by schmitzoide** [comment]
> Can you delete the injection project

How do I do that?

**Comment by johnno1962** [comment]
If you send your TeamViewer co-ord to injectionapp at johnholdsworth.com I can take a look

**Comment by sibljon** [comment]
I'm experiencing the same issue (first injection works, subsequent fail with a `no suitable image found` error), but I'm able to resolve it by `rm -rf ./iOSInjectionProject` within the `.../DerivedData/.../Logs/Build/...` directory that is printed in the `Injection Console`. @johnno1962 this is what you were asking @schmitzoide to try, right?

However, I'm not sure that this is a complete solution, as I am not able to get injection fully working as I subsequently experience an issue which I've detailed in #10 after the swizzling stage.

**Comment by schmitzoide** [comment]
That was it yes @sibljon, I'll try to do what you've done. But do you do it all the time that you inject?

**Comment by sibljon** [comment]
Yes, unfortunately I have to manually delete the directory every time I
inject :-/
